### PR TITLE
fix crash 

### PR DESCRIPTION
**Pull request by tfkbudi**
now..in sample app..the app will crash if you enable add silent and add setCurrentRingtone. That because when any value ini currentRingtone you will loop but silent uri is null. then will make the app get NPE.

### CODE REVIEW

File: ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java [hunk 1]
```diff
@@ -257,7 +257,7 @@ private void prepareRingtoneList(final ArrayList<Integer> types) {
 
         //Add silent item first
         if (isDisplaySilent) {
-            mRingTones.put(getString(R.string.title_silent_list_item), null /* No ringtone */);
+            mRingTones.put(getString(R.string.title_silent_list_item), Uri.parse("") /* No ringtone */);
```
Comment (kevalpatel2106) [inline ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java]: Can you use ‘URI.EMPTY’ instead of ‘URI.parse()’?

**Review by kevalpatel2106 — CHANGES_REQUESTED** [review]
Just one change. Everything else looks good.👍

File: ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java (hunk 1 above)
Comment (tfkbudi) [reply ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java]: hi...i already changed the code. now using Uri.EMPTY ...

**Comment by kevalpatel2106** [comment]
Thanks for your change @tfkbudi. You're awesome. 👍

**Pull request merged**